### ISSUE TITLE
External link checker is confusing on blocks with multiple links

### ISSUE THREAD

**Issue by cbirdsong** [itmaybejj/editoria11y-wp#7]
It's a little confusing to have the whole block marked for review when it has multiple links, only one of which is external:

![CleanShot 2023-04-20 at 08 39 39](https://user-images.githubusercontent.com/1672206/233384238-020230aa-8fb2-4c90-8735-7740d140039b.png)

It might be better if the message was anchored to the links themselves?

![CleanShot 2023-04-20 at 08 49 59](https://user-images.githubusercontent.com/1672206/233387130-e4ad8141-57f8-48ae-935c-79e3e66bbf20.png)

Some rough CSS - it likely would need to be scoped to paragraphs/headings/lists and not _every_ link:
```css
.editor-styles-wrapper :not(.is-selected) a[target="_blank"] {
    position: relative;
    outline: 1px solid #fad859;
    outline-offset: 0;
    box-shadow: 0 0 0 2px #fad859;
    border-radius: 2px;
}

.editor-styles-wrapper :not(.is-selected) a[target*="_blank"]::after {
    content: "Manual check: is opening a new window expected?";
    position: absolute !important;
    bottom: 100% !important;
    left: -2px !important;
    width: auto !important;
    height: auto !important;
    background: #fad859 !important;
    color: #111 !important;
    display: inline-block !important;
    padding: 4px 4px 2px 6px !important;
    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif !important;
    font-weight: 500 !important;
    font-size: 13px !important;
    line-height: 1 !important;
    border-radius: 2px 0 0 0 !important;
    letter-spacing: 0 !important;
    z-index: 5 !important;
    pointer-events: none !important;
}
```

**Comment by itmaybejj** [comment]
Very much agreed. That's what I would like to do. I'm a little worried about conflicting with theme CSS though if I start modifying internal elements -- e.g., external link icons.

I think in the long run I need to implement it to work like the normal frontend checker -- just visually positioning toggles near elements. The challenge is doing that without modifying the DOM inside the block editor. I could generate my toggles at the end of the page and then position them absolutely relative to the page, which would work, at the cost of keyboard accessibility. But I'll play with your code and ponder in the interim.

**Comment by cbirdsong** [comment]
Personally, if I had a theme using external link icons I would probably be fine having them overridden by this tool in the editor?

**Comment by itmaybejj** [comment]
True. The only real excitement would be a partial replacement -- e.g., my content getting inserted, but having it bound by a max-width and clipped off or whatever. But that would be a bug for future me.

I'll have to experiment. I'm confident I can make it work, and I should be able to throw time at it this summer.

**Comment by itmaybejj** [comment]
Fixed in [1.0.13!](https://github.com/itmaybejj/editoria11y-wp/releases/tag/1.0.13)
